### ISSUE TITLE
options: allow to pass a webpack compatible configuration object

### ISSUE THREAD

**Issue by alex3683** [LaxarJS/laxar-infrastructure#10]
Currently laxar-infrastructure only takes some selected keys from the `options` that are passed to it, and sets those on the resulting webpack configuration object.
There are some issues with this:
1. Its hard to understand what is taken from the options and what is ignored (okay, documentation is still missing). The expectation is, that I can provide a (partial) webpack configuration and get a working webpack configuration as a result.
2. I just now need to set `resolve.extensions` and this belongs to the ignored options ;-)

I'd favor an API that is more intuitive. 🤓

**Comment by jpommerening** [comment]
Yes, let's do this!

I always like to sneak in some…

### BREAKING CHANGE

If you used the `rules` or `alias` options before, you will now need to wrap them in their proper parent property the way webpack would expect them.

#### Before
```js
laxarInfrastructure.webpack( {
   context: ___dirname,
   alias: { x: y },
   rules: [
      { test: /\.meh$/, use: 'meh-loader' }
   ]
} );
```

#### After:

```js
laxarInfrastructure.webpack( {
   context: ___dirname,
   resolve: {
      // you can now use resolve.extensions, resolve.browserFields, etc.!
      alias: { x: y }
   },
   module: {
      rules: [
         { test: /\.meh$/, use: 'meh-loader' }
      ]
   }
} );
```

**Comment by jpommerening** [comment]
released as `v0.1.0-alpha.6`

**Comment by jpommerening** [comment]
Probably affected repos: https://github.com/search?utf8=✓&q=laxar-infrastructure+rules+user%3ALaxarJS+path%3A%2F+extension%3Ajs&type=Code 😭
